src/common/react/named-sfc.ts

```diff
@@ -2,10 +2,10 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-export type ReactSFCWithDisplayName<P = {}> = React.SFC<P> & { displayName: string };
+export type ReactFCWithDisplayName<P = {}> = React.FC<P> & { displayName: string };
 
-export function NamedSFC<P = {}>(displayName: string, component: React.SFC<P>): ReactSFCWithDisplayName<P> {
+export function NamedSFC<P = {}>(displayName: string, component: React.FC<P>): ReactFCWithDisplayName<P> {
```

Comment:
It's a broader change (130+ files) but do we want to consider renaming `NamedSFC` as well?

Reply:
I missed that (searched for react.sfc only)... I can do it as part of this PR or a separated one, any preference @peterdur ?

Reply:
done